world.js

```diff
@@ -110,12 +111,24 @@ World.prototype.locate = function(point) {
   return ind
 }
 
+World.prototype.cues = function() {
+  var cues = []
+  this.tiles.forEach(function (tile) {
+    var cue = _.find(tile.children, function(child) {return child.props.cue})
```

Comment:
Currently only children, I think I'd rather and wait and solve the recursive thing nicely for both `cue` and `obstacle` properties, whenever we need it.